src/routes/tasks/components/TaskIOPermissions.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Icon } from '@chakra-ui/react';
+import { RiGroupLine } from 'react-icons/ri';
+
+import { PermissionsT } from '@/types/CommonTypes';
+
+import { TaskIOTooltip } from '../TasksUtils';
+
+const TaskIOPermissions = ({
+    permissions,
+}: {
+    permissions?: PermissionsT | null;
```

Comment:
Not mandatory: instead of defining the prop types inline, you could define them separately for better readability and reusability if more come:
```
type TaskIOPermissionsProps = {
    permissions?: PermissionsT | null;
};
```

Reply:
True. I made the choice to not declare a type as it is only used here and it only contains permissions 👍